compiler/rustc_error_codes/src/error_codes/E0206.md

```diff
@@ -1,5 +1,5 @@
 The `Copy` trait was implemented on a type which is neither a struct nor an
-enum.
+enum nor a union.
```

Comment:
This duplicates a "nor", I think this should be "is neither a struct, an enum, nor a union".